vmm/task/src/container.rs

```diff
@@ -118,7 +118,7 @@ impl ContainerFactory<KuasarContainer> for KuasarFactory {
         req: &CreateTaskRequest,
     ) -> containerd_shim::Result<KuasarContainer> {
         rescan_pci_bus().await?;
-        let bundle = format!("{}/{}", KUASAR_STATE_DIR, req.id);
+        let bundle = format!("{}{}/{}", KUASAR_STATE_DIR, req.id, "bundle");
```

Comment:
Because KUASAR_STATE_DIR is defined as "/run/kuasar/state/", with the last '/', so i deleted '/' here......
But maybe it should not ended with ‘/’, I'll optimize it together.

Reply:
fixed @Burning1020